packages/nimble/inst/tests/test-benchmark-building-steps.R

```diff
@@ -45,24 +45,40 @@ test_that('Benchmarking model and MCMC building and compiling steps',
 {
     caseNames <- character()
 
-    caseNames[1] <- 'theta->mu[1:1000]->y[1:1000]'
+    ## 1000 is a good size for full benchmarking
+    Benchmark1length <- 10
```

Comment:
Do I understand correctly that your typical workflow is to modify this code and run from within emacs?